From 912d97dc96b03473cba3039aa2757ccce8e7b47d Mon Sep 17 00:00:00 2001
From: Zheng Zengkai <zhengzengkai@huawei.com>
Date: Tue, 6 Jul 2021 12:10:21 +0800
Subject: [PATCH] config: disable config ARM64_BOOTPARAM_HOTPLUG_CPU0 by
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 912d97dc96b03473cba3039aa2757ccce8e7b47d

 default

hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I3Z80Y
CVE: NA

-------------------------------------------------

Disable config ARM64_BOOTPARAM_HOTPLUG_CPU0 in openeuler_defconfig
by default.

Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Reviewed-by: Hanjun Guo <guohanjun@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 07b4082459ac..1b442b1d129b 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -378,6 +378,7 @@ CONFIG_SCHED_MC=y
 CONFIG_SCHED_SMT=y
 CONFIG_NR_CPUS=1024
 CONFIG_HOTPLUG_CPU=y
+# CONFIG_ARM64_BOOTPARAM_HOTPLUG_CPU0 is not set
 CONFIG_NUMA=y
 CONFIG_NODES_SHIFT=4
 CONFIG_USE_PERCPU_NUMA_NODE_ID=y
-- 
2.26.2

